topics/search.md

```diff
@@ -0,0 +1,49 @@
+# Valkey-Search
+
+**Valkey-Search** (BSD-3-Clause), provided as a Valkey module, is a high-performance Vector Similarity Search engine optimized for AI-driven workloads. It delivers single-digit millisecond latency and high QPS, capable of handling billions of vectors with over 99% recall.
+
+Valkey-Search allows users to create indexes and perform similarity searches, incorporating complex filters. It supports Approximate Nearest Neighbor (ANN) search with HNSW and exact matching using K-Nearest Neighbors (KNN). Users can index data using either **Valkey Hash** or **[Valkey-JSON](https://github.com/valkey-io/valkey-json)** data types.
+
+While Valkey-Search currently focuses on Vector Search, its goal is to extend Valkey into a full-fledged search engine, supporting Full Text Search and additional indexing options.
+
+## Supported Commands
+
+```plaintext
+FT.CREATE
+FT.DROPINDEX
+FT.INFO
+FT._LIST
+FT.SEARCH
+```
+
+For a detailed description of the supported commands and configuration options, see the [Command Reference](https://github.com/valkey-io/valkey-search/blob/main/COMMANDS.md).
```

Comment:
Stay within the website, we should link to valkey.io/commands/#search presumably.